src/main/java/no/sikt/generator/ApiData.java

```diff
@@ -24,10 +23,10 @@ public class ApiData {
 
     private final RestApi awsRestApi;
     private final OpenAPI openapiApiGateway;
-    private Optional<OpenAPI> openapiApiGithub;
```

Comment:
Optional fields are a no-no, I replaced this with a private getter…it serves little real purpose aside hiding null checks. I suspect we simply have a suboptimal design here (typically, we see this kind of thing when the OO is off), which should probably be rectified.